### ISSUE TITLE
Exscape Scopes in Regex

### ISSUE THREAD

**Issue by rdebeasi** [obahareth/are-you-es5#20]
When `are-you-es5` finds a namespaced package that is not ES5, the resulting regex has an unescaped forward slash:

```
/[\\/]node_modules[\\/](?!(@react-pdf/renderer|dotenv|md5-file|pino|roboto-fontface)[\\/])/
```

The regex should be:
```
/[\\/]node_modules[\\/](?!(@react-pdf\/renderer|dotenv|md5-file|pino|roboto-fontface)[\\/])/
```

Thanks for making this awesome tool! :)

**Comment by obahareth** [comment]
@rdebeasi Can you check if the newly published version `1.3.3` fixes this for you?

**Comment by rdebeasi** [comment]
1.3.3 does indeed fix the issue. Thank you!!
